components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -173,6 +181,23 @@ class AtMentionSuggestion extends Suggestion {
             );
         }
 
+        let countBadge;
+        if (item.type === Constants.MENTION_GROUPS) {
+            countBadge = (
+                <span className='suggestion-list__group-count'>
+                    <Badge>
+                        {(item as Group).member_count}
+                        <span className='ml-1'>
+                            <FormattedMessage
```

Comment:
We should do something like this to surface the count using `i18n`:
```
<FormattedMessage
     id='user_groups_modal.memberCount'
     defaultMessage='{member_count} {member_count, plural, one {member} other {members}}'
      values={{
          member_count: group.member_count,
     }}
 />
```

Reply:
@BenCookie95 I wasn't aware there was such functionality, thanks!